test/apps/iodemo/io_demo.cc

```diff
@@ -2822,7 +2837,7 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
             }
             break;
         case 'I':
-            test_opts->src_addr = optarg;
+            set_src_addrs(optarg, test_opts);
```

Comment:
maybe -I could be passed multiple times. every -I argument adds another address. then, no need to parse by ,

Reply:
Good idea. This could simplify implementation and minimize change.

Reply:
Done